### PR TITLE
This appears to be way to control the generated module namespace for ruby

### PR DESCRIPTION
**Pull request by tmeasday**
See https://github.com/google/protobuf/blob/master/ruby/tests/generated_code.proto#L3

### CODE REVIEW

File: proto/reports.proto [hunk 1]
```diff
@@ -28,6 +28,8 @@ import "google/protobuf/descriptor.proto";
 option java_package = "apollo.optics.proto";
 option optimize_for = SPEED;
 
+package optics_agent.proto;
```
Comment (pcarrier) [inline proto/reports.proto]: Could you move this above the options, and pick something that doesn't contain the word agent? The same protobuf should be used across the pipeline so it's not agent-specific.

**Comment by pcarrier** [comment]
Actually, why don't just get rid of `option java_package` and use `package apollo.optics.proto` for everybody?

**Comment by tmeasday** [comment]
How about now @pcarrier?

**Comment by pcarrier** [comment]
Thanks!

**Pull request merged**